<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="mytitlewidget" name="titlewidget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1677215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="login_page">
      <widget class="QWidget" name="total_widget" native="true">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>501</width>
         <height>232</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>华文行楷</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>用户登录</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/ico/images/password.PNG&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_login_pass</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/ico/images/username.PNG&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_login_username</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEdit_login_pass"/>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lineEdit_login_username">
               <property name="font">
                <font>
                 <family>Agency FB</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="checkBox_login_remember">
               <property name="text">
                <string>记住密码</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="reg_Button">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                 <underline>true</underline>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 0, 0);</string>
               </property>
               <property name="text">
                <string>没有账户，立即注册</string>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_login">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>20</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Agency FB</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/ico/images/balckButton.png);
color: rgb(0, 85, 255);
</string>
               </property>
               <property name="text">
                <string>登录</string>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="register_page">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>681</width>
         <height>428</height>
        </rect>
       </property>
       <widget class="QLabel" name="reg_title_label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>华文新魏</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>欢迎注册</string>
        </property>
       </widget>
       <widget class="QToolButton" name="allright_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>128</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>已有账号？登录</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_reg">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>340</y>
          <width>400</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>方正姚体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/ico/images/balckButton.png);
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>481</width>
          <height>294</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>用户名：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_username">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="username_error_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
font: 75 8pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>昵称：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_nickname</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_nickname">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="Nickname_error_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
font: 75 8pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>密码：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_password</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="lineEdit_password">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="password_error_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
font: 75 8pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>确认密码：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_confirmpass</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="lineEdit_confirmpass">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="passconfirm_error_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
font: 75 8pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>手机号：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_phonenum</cstring>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLineEdit" name="lineEdit_phonenum">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="phonenum_error_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
font: 75 8pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>邮箱：</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_email</cstring>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QLineEdit" name="lineEdit_email">
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QLabel" name="email_error_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
font: 75 8pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="set_page">
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>531</width>
         <height>351</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>96</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>网络设置</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_net_confirm">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>260</y>
          <width>231</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
border-image: url(:/ico/images/balckButton.png);</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="IP_error_label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="port_error_label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>220</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_set_port">
        <property name="geometry">
         <rect>
          <x>226</x>
          <y>196</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>131</x>
          <y>141</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>IP地址：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_set_IP</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>131</x>
          <y>196</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>端口号：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_set_port</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_set_IP">
        <property name="geometry">
         <rect>
          <x>226</x>
          <y>141</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mytitlewidget</class>
   <extends>QWidget</extends>
   <header location="global">mytitlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_login_username</tabstop>
  <tabstop>lineEdit_login_pass</tabstop>
  <tabstop>checkBox_login_remember</tabstop>
  <tabstop>reg_Button</tabstop>
  <tabstop>toolButton_login</tabstop>
  <tabstop>allright_Button</tabstop>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_nickname</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_confirmpass</tabstop>
  <tabstop>lineEdit_phonenum</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>toolButton_reg</tabstop>
  <tabstop>lineEdit_set_IP</tabstop>
  <tabstop>lineEdit_set_port</tabstop>
  <tabstop>toolButton_net_confirm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
